composer.json

```diff
@@ -28,7 +28,7 @@
         "symfony/options-resolver": "^3.0 || ^4.0",
         "symfony/polyfill-php70": "^1.0",
         "symfony/polyfill-php72": "^1.4",
-        "symfony/process": "^3.0 || ^4.0",
+        "symfony/process": "^3.0 || ^4.0 <4.2",
```

Comment:
with the other changes in the PR would it be possible to remove this restriction and run with SF 4.2 packages?

Reply:
still `keradus/cli-executor` would make SF 4.2 process deprecation to happen